wrappers/tcpmib.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2014-2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the license is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package wrappers
+
+const (
+	MIB_TCP_STATE_CLOSED     = 1
+	MIB_TCP_STATE_LISTEN     = 2
+	MIB_TCP_STATE_SYN_SENT   = 3
+	MIB_TCP_STATE_SYN_RCVD   = 4
+	MIB_TCP_STATE_ESTAB      = 5
+	MIB_TCP_STATE_FIN_WAIT1  = 6
+	MIB_TCP_STATE_FIN_WAIT2  = 7
+	MIB_TCP_STATE_CLOSE_WAIT = 8
+	MIB_TCP_STATE_CLOSING    = 9
+	MIB_TCP_STATE_LAST_ACK   = 10
+	MIB_TCP_STATE_TIME_WAIT  = 11
+	MIB_TCP_STATE_DELETE_TCB = 12
+)
+
+type MIB_TCPROW struct {
+	State      uint32
+	LocalAddr  uint32
+	LocalPort  uint32
+	RemoteAddr uint32
+	RemotePort uint32
+}
+
+type MIB_TCPTABLE struct {
+	NumEntries uint32
+	//Table    []MIB_TCPROW
```

Comment:
Considering adding a comment explaining you cast raw buffers instead to walk the array.